app/src/assets/utilities/index.js

```
@@ -4,11 +4,12 @@
 
 const IMAGES_DIR = '../images/';
 const appImages = {
-  logo: require(IMAGES_DIR + 'logowithwhiteBG.png'),
+  logo: require(IMAGES_DIR + 'logo/transblack.png'),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)